docs/keywords.md

```diff
@@ -86,13 +86,13 @@ ajv.addKeyword({
   code(cxt: KeywordCxt) {
     const {schema, parentSchema, data} = cxt
     const [min, max] = schema
-    const eq: Code = parentSchema.exclusiveRange ? _`=` : nil
-    gen.fail(_`${data} <${eq} ${min} || ${data} >${eq} ${max}`)
+    const eq: Code = parentSchema.exclusiveRange ? _`=` : null
+    cxt.fail(_`${data} <${eq} ${min} || ${data} >${eq} ${max}`)
```

Comment:
just wrondering, isn't `ctx` more common abbreviation for 'context'? that's what we use internally

Reply:
I've seen both used a lot, lately `cxt` was more common around me :)...